terraform/shared/video-player/js/common.js

```diff
@@ -1,6 +1,5 @@
 // IPFS config
-var ipfs_gateway_self = '__IPFS_GATEWAY_SELF__';     // IPFS gateway of this node
-var ipfs_gateway_origin = '__IPFS_GATEWAY_ORIGIN__'; // IPFS gateway of origin stream

```

Comment:
You probably need to grep for `__IPFS_GATEWAY_ORIGIN__` and remove references in the terraform provisioning scripts?

Reply:
Issue #90  I would like to flatten all the PRs first